libraries/chain/backing_store/db_context_rocksdb.cpp

```diff
@@ -288,19 +291,12 @@ namespace eosio { namespace chain { namespace backing_store {
 
    int32_t db_context_rocksdb::db_get_i64(int32_t itr, char* value , size_t value_size) {
       const auto& key_store = primary_iter_store.get(itr);
```

Comment:
Is it possible that itr is not cached in the iter_store?

Reply:
Only if other code was written incorrectly. the itr is a "handle" to the code/scope/table/primary_key that would have been stored and returned by a call to db_find_i64 (or db_lower/upper_bound, db_next/previous_i64, etc)